src/popup/components/Modals/AccountImport.vue

```diff
@@ -83,21 +86,22 @@ export default defineComponent({
         .trim();
 
       if (!validateSeedLength(mnemonicParsed)) {
-        error.value = root.$t('pages.index.invalidSeed');
+        error.value = t('pages.index.invalidSeed');
         return;
       }
       if (!mnemonicParsed || !validateMnemonic(mnemonicParsed)) {
-        error.value = root.$t('pages.index.accountNotFound');
+        error.value = t('pages.index.accountNotFound');
         return;
       }
-      root.$store.commit('setMnemonic', mnemonicParsed);
-      root.$store.commit('setBackedUpSeed');
+      store.commit('setMnemonic', mnemonicParsed);
+      store.commit('setBackedUpSeed');
       props.resolve();
       setTimeout(async () => {
-        await getSdk();
```

Comment:
Does other places where we await `getSdk()` works correctly? Was this issue only related to this particular place?

Reply:
As far I checked (for example balances update) it seems to works fine and it was faulty in this place only.